blingfire/src/lib.rs

```diff
@@ -125,7 +122,7 @@ mod tests {
 
     #[test]
     fn text_to_words_string_too_long() {
-        let source = String::from_utf8(vec![b'.'; MAX_SOURCE_LENGTH + 1]).unwrap();
+        let source = String::from_utf8(vec![b'.'; MAX_TEXT_LENGTH + 1]).unwrap();
         let mut destination = String::new();
         assert!(text_to_words(&source, &mut destination).is_err());
```

Comment:
also, i know this was pre-existing but could you check that the *actual* error is `SourceTooLong`

Reply:
done